USWDS-Compile - Sass: Create quiet deprecations setting (#131)
uswds/uswds-compile: README.md

| --------------------------------- | -------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| `sprite.projectIconsOnly`         | `false`                                            | Include _only_ the icons in `paths.src.projectIcons` in the icon sprite.                                                                                                                        |
| `settings.compile.browserslist`   | `["> 2%", "last 2 versions", "IE 11", "not dead"]` | Define the [browserslist query](https://github.com/browserslist/browserslist?tab=readme-ov-file#queries) for CSS prefixes generated by [autoprefixer](https://github.com/postcss/autoprefixer).      |
| `settings.compile.sassSourcemaps` | `true`                                             | Include sourcemap when compiling SASS to CSS.                                                                                                                                                   |

> | `settings.compile.quietSassDeps`  | `true`                                             | Silence USWDS Sass deprecation warnings. Your custom project Sass/SCSS will still trigger warnings if necessary.                                                                                |

Reviewer: **suggestion:** Proposing a small tweak for clarity. 
```suggestion
| `settings.compile.quietSassDeps`  | `true`                                             | Silence USWDS Sass deprecation warnings. This will not silence deprecation warnings for project Sass.                                                                                |
```
